wordpress-mobile/WordPress-Shared-iOS
# Add iPhone Plus device identification

Our existing `WPDeviceIdentification` support for Plus sized iPhones only checks for specifically the iPhone 6 Plus (using its device name). I've added a new method to check for Plus sized phones (unzoomed), which is needed in WPiOS to customise split views slightly for these devices.

I've also added a test to the Device Identification section of the example project.

To test: 

* Open up the example project and check that the 'iPhone Plus (Any)' row shows a YES result on Plus devices. 
* Please test on a real device and on the Simulator.
* Note that the existing `isiPhoneSixPlus` method returns `YES` for any Plus device on the Simulator, but only for an actual _6+_ on devices. The new method should work for any Plus device.
* Turn on display zoom in your device's settings and check that the method now returns NO.

Needs review: @nheagy
